utils/obsproc/IcecAbi2Ioda.h

```diff
@@ -0,0 +1,179 @@
+#pragma once
+
+#include <complex>
+#include <ctime>
+#include <iomanip>
+#include <iostream>
+#include <map>
+#include <netcdf>    // NOLINT (using C API)
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "eckit/config/LocalConfiguration.h"
+
+#include <Eigen/Dense>    // NOLINT
+
+#include "ioda/../../../../core/IodaUtils.h"

```

Comment:
Not related to this PR, but we need to fix this at some point. Can you add a TODO here?

Reply:
Added